lib/JiraClient.ts

```diff
@@ -28,7 +28,13 @@ export class JiraClient {
       }
     }
 
-    public async findTransition(issueId: string, transitionName: string): Promise<any> {
+    public async transitionIssue(issueId: string, transition: any): Promise<void> {
```

Comment:
Hmm, this one also doesn't need to be public anymore. And then, you can swap the method back to avoid confusing the git diff :)
```suggestion
    private async transitionIssue(issueId: string, transition: any): Promise<void> {
```